packages/react-scripts/config/webpack.config.prod.js

```diff
@@ -65,6 +66,8 @@ const customModuleRegexes = bpkReactScriptsConfig.babelIncludePrefixes
   : [];
 
 const optInCssModules = bpkReactScriptsConfig.cssModules === false;
+const optimizeCssOptions = bpkReactScriptsConfig.optimizeCssOptions ? bpkReactScriptsConfig.optimizeCssOptions : {}
```

Comment:
I figured we'd ship it silently to futureproof when someone inevitably finds something they want to tweak which is why I didn't include docs